content/style_guide.md

```diff
@@ -565,14 +587,89 @@ like "Create a new user" (by its very nature, a user created **is** a
 new user) or (for a setting named `cidr_block`) "The CIDR block for the
 VPC."
 
+## Examples
+
+### Example Domains
+
+Use the `example.com` domain for generic domains and email addresses in the documentation. Use the `chef.io` or `progress.com` domain for examples that should refer back to Chef Software.
+
+### Example Names
+
+Don't reveal personal information in examples, such as the names of real people, real email addresses, or phone numbers.
+
+Don't use the names of bands, musicians, or characters from works that are under copyright.
+
+When writing about security, follow the accepted convention and use "Alice" and "Bob". Following this convention helps readers see that they are reading a topic about security and integrate the Chef information with their existing knowledge.
+
+Here is a list of some example names for you to use (all of the last names are translations of "Chef", according to the internet):
+
+Ares Koch
+Tamira Bucatar
+Dan Gotvach
+Lena Chushi
+Haris Shefu
+Booker Yolisa
+Kala Baavarchee
+Samuel Tagaluto
+
+### Example Email Addresses
+
+akoch@example.com
+tvucatar@example.com
+dgotvach@example.com
+lchushi@example.com
+hshefu@example.com
+byolisa@example.com
+kbaavarchee@example.com
+stagaluto@example.com
+
+Use `docs@chef.io` for the Chef Technical Documentation team.
+
+#### Example Phone Numbers
+
+Never use a real phone number in an example. For a US phone number, use one from the range reserved for examples in fiction, which is (800) 555-0100 through (800) 555-0199.
+
+### Example Addresses
+
+Use fictional street addresses in examples.
+
+Use these fictional addresses:
+
+2943 Conifer Drive
+Seattle, WA 98122
+
+1214 Hollow Road
+Boston, MA 02110
```

Comment:
It turns out, adding two spaces after a line followed by another line keeps the two lines in the same paragraph, but adds a <br> between the lines. Who knew.

```suggestion
2943 Conifer Drive  
Seattle, WA 98122

1214 Hollow Road  
Boston, MA 02110
```

Reply:
The wonders of Hugo.